lib/resources.js

```diff
@@ -276,23 +276,48 @@ module.exports = function (app, _private = null) {
       .then((items) => ItemResultsSerializer.serialize(items, opts))
   }
 
+  const addInnerHits = (body) => {
+    body.query.function_score.query.bool.should.push({
+      nested: {
+        path: 'items',
+        query: { 'match_all': {} },
+        inner_hits: {
+          sort: [{ 'shelfMark_sort': 'asc' }],
+          size: process.env.SEARCH_ITEMS_SIZE
```

Comment:
What does this default to if it is not set? Should we add it to `.env.example`?

Reply:
Good point. Added a default, also added to `.env.example`